heliopy/util/config.py

```diff
@@ -49,18 +49,18 @@ def load_config():
     """
     config_location = get_config_file()
     config = configparser.ConfigParser()
-    config.read(config_location)
+    config.read(str(config_location))
     config_dict = {}
 
     # Set data download directory
-    download_dir = os.path.expanduser(config['DEFAULT']['download_dir'])
+    download_dir = Path(config['DEFAULT']['download_dir']).expanduser()
     if os.name == 'nt':
```

Comment:
You can remove this condition since you're using pathlib! Pathlib does it automatically.